### DISCUSSION TITLE
Making the style to be exactly the same seems to not give perfect match score and the colors given are not the one's shown in Target #3

### DISCUSSION THREAD

**Discussion by akkshaylawrence** [cssbattle/forum#62] [Bugs reports]
In [#3 - Push Button](https://cssbattle.dev/play/3) even though the design is perfectly matching the score given is not 100%. I suspect that this issue is caused because the given colors are not matching the given solution image colors as per the color picker. See the image below to inspect the issue.

The three colors _**according to the solution image**_ should be: **#6f91ca**, **#243d83** **#e6ba62** respectively according to my color picker, but given colors are #6592CF #243D83 and #EEB850. using the color picked colors will further decrease the score as it does not match the given colors.

<img width="1440" alt="Screenshot 2021-05-31 at 1 54 37 PM" src="https://user-images.githubusercontent.com/12969853/120164058-c7f9b800-c217-11eb-919f-b957de873071.png">

This is on Firefox 88.0.1 on an M1 MacBook Pro

**Comment by chinchang** [comment]
Hey @akkshaylawrence You should use a webkit browser like Google Chrome to play because that is what is used at the backend evaluation. Other browsers might render output differently

**Reply by akkshaylawrence** [reply]
@chinchang Great! that actually fixed the issue 👍🏼
